pinot-common/src/main/java/org/apache/pinot/common/utils/DataSchema.java

```diff
@@ -277,6 +277,10 @@ public ColumnDataType getStoredType() {
           return LONG;
         case JSON:
           return STRING;
+        case BOOLEAN_ARRAY:
```

Comment:
What about BigDecimal ?

Reply:
BigDecimal seems to be stored as is in its binary format. unless I am mistaken @Jackie-Jiang